### ISSUE TITLE
Ether not shown in wallet (or any wallet) 

### ISSUE THREAD

**Issue by CalvinHartwell** [ethereum/mist#3687]
```
Version: 0.9.3
Ethereum Wallet: 0.9.3
OS & Version: windows 10 OR Ubuntu 17.04 or 17.10
Node version: geth 1.8.0 beta or 1.7.X stable.  
Number of blocks synchronized: 5091511
```

Hi all, 

I'm basically hitting this problem: 

- Ether is not shown in the wallet

I've tried geth, mist and ethereum wallet on Windows and Ubuntu on three different machines in three continents and I can't get my balance to appear. According to etherscan, the balance exists: [https://etherscan.io/address/0x89638edc154c49BD71544775b3b8e186188c036c](https://etherscan.io/address/0x89638edc154c49BD71544775b3b8e186188c036c)

Any ideas? This was a subwallet of my main wallet. 

I'm surprised at how bad this is, I've already made multiple transactions using Monero, its very easy. 

Thanks for help! 

 -->

**Comment by CalvinHartwell** [comment]
I ended up sending via myetherwallet, but this is pretty weird. I've tried so many times to make this work, it seems like the latest build is very broken. Is this caused by network contention?

**Comment by FGYPXZ** [comment]
This is just bullshit. How can it cost so much if the wallet does not work properly? Ethereum is a bubble and overestimated. The second cryptocurrency in the world with absolutely useless wallet.

**Comment by simon66** [comment]
@FGYPXZ Or maybe is the user that doesn't know how to use it properly 👍 

@CalvinHartwell there seems to be an issue with the latest Mist wallet where it will fail to sync with the network. You will need to fully sync with the network in order to see your balance. There is the `light client` [option in Mist](https://steemit-production-imageproxy-thumbnail.s3.amazonaws.com/U5drXXCUPeq7BdNqSjpo6uvwdU4ceuz_1680x8400), but I've never used it ). Your best bet is to use a hardware wallet, or MetaMask.

**Comment by marcgarreau** [comment]
This may fall into a category of issues resolved with the [0.11.0 release](https://github.com/ethereum/mist/releases/tag/v0.11.0). Please give that a try and open a new issue if appropriate.
